docs/02-guides/flash-messages.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Flash messages
+description:
+  Learn how to push and display flash messages in your Front-Commerce
+  application.
+---
+
+<SinceVersion tag="3.6" />
+
+<p>{frontMatter.description}</p>
+
+## When to use flash messages?
+
+Flash messages are used to display informations to the user. These messages are
+meant to be displayed once and are stored in the session. When consummed, they
+are removed from the session.
+
+## Flash message data type
+
+Flash messages are represented by an object the following properties:
+
+| Property  | Type   | Description                                                                                                                    |
+| --------- | ------ | ------------------------------------------------------------------------------------------------------------------------------ |
+| `type`    | string | The type of the flash message. It can be one of the types declared in the [flash messages component](#flash-messages-component). |
+| `message` | string | The message to display.                                                                                                        |
+| `data`    | object | An object that can be used to pass additional data to the flash message.                                                       |
+
+## Flash messages component
+
+Theme chocolatine provides default flash messages component depending on the
+`type` property of the flash message object.
+
+It can be customized by overriding the
+[`theme/modules/FlashMessages/getFlashMessageComponent.tsx`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/9cc7010e5d3c3ade454cdeaad90f28c769916695/packages/theme-chocolatine/theme/modules/FlashMessages/getFlashMessageComponent.jsx)
+file.
+
+## Adding flash messages
+
+Flash messages can be added in your code with the FrontCommerce application
+instance. Here's an example on how to do it in a route loader:
```

Comment:
**nitpick:** it can be done from the session. The `FrontCommerceApp` is only one of the way to access it (e.g: resolvers can use the `context.user` object for instance).

**suggestion:** reword the sentence

```suggestion
Flash messages can be added in your code from the user session object. Here's an example on how to do it in a route loader via the `FrontCommerceApp` instance:
```

Reply:
fixed with your suggestion